sdk/src/feature_set.rs

```diff
@@ -826,6 +830,7 @@ lazy_static! {
         (vote_state_add_vote_latency::id(), "replace Lockout with LandedVote (including vote latency) in vote state #31264"),
         (checked_arithmetic_in_fee_validation::id(), "checked arithmetic in fee validation #31273"),
         (bpf_account_data_direct_mapping::id(), "use memory regions to map account data into the rbpf vm instead of copying the data"),
+        (round_compute_unit_price::id(), "round down compute-unit-price to the nearest lamports #31453"),
```

Comment:
```suggestion
        (round_compute_unit_price::id(), "round down compute-unit-price to the nearest milli-lamport #31453"),
```
or do we not want to start using milli-lamports at all 🫠
either way the current feature text is out of date

Reply:
yea, was intentionally trying not to add another term, updating feature text with "1_000 micro-lamport" to be consistent with other places. Thanks for spotting this.

Reply:
millilamports has too many "l"s and "i"s next to eachother for my brain to like, thanks for not adding this as an official unit :pray: